src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -55,8 +55,11 @@ public Question nextQuestion(@PathVariable("id") String gameId){
     }
 
     @PostMapping("/checkGuess")
-    public GuessResult checkGuess(@RequestBody Guess userGuess){
-        GuessResult result = gameService.ProcessPlayerGuess(userGuess.getGuess(),userGuess.getGameCode(), userGuess.getQuestionId(), userGuess.getPlayerName());
+    public GuessResult checkGuess(@RequestBody Guess userGuess) throws Exception {
+        GuessResult result = gameService.ProcessPlayerGuess(userGuess.getGuess()
+                ,userGuess.getGameCode()
+                , userGuess.getQuestionId()
+                , userGuess.getPlayerName());
```

Comment:
reformatted the arguments to use indentation so the code may be cleaner and easier to read.
This recommended by the "methods" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3